### PR TITLE
UnitySDK: Update parameters.

### PR DESCRIPTION
**Pull request by hartleysarah72**

### CODE REVIEW

File: UnitySDK/.vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (lauri-h) [inline UnitySDK/.vscode/extensions.json:1]: Can you add the .vscode folder to .gitignore?

**Review by lauri-h — APPROVED** [review]

File: UnitySDK/.vscode/extensions.json (hunk 1 above)
Comment (hartleysarah72) [reply UnitySDK/.vscode/extensions.json:1]: I'll add it. Should I remove that folder from these changes too?

Comment (lauri-h) [reply UnitySDK/.vscode/extensions.json:1]: Once you update it they won't be tracked to git. You should do that before you merge

File: UnitySDK/Assets/PlayerWon-SDK/Classes/ClientDetails.cs [hunk 2]
```diff
@@ -61,8 +61,24 @@ public class ClientDetails
         [SerializeField]
         private int gs;
 
-        private ClientDetails(string _country, string _idfa, string _plat, string _lang, string _ip = null, int _w = 0, int _h = 0, string _pid = null, string _pidType = null, string _device = null, int _coppa = 0,
-            int _underage = 0, int _age = 0, string _slot_id = null, int _ls = 0, Currency _lsv = null, CurrencyArray _rv = null, int _gs = 0)
+        [SerializeField]
+        private int lt;
```
Comment (victorhurdugaci) [inline UnitySDK/Assets/PlayerWon-SDK/Classes/ClientDetails.cs:65]: nit but might affect how this is used. If this field is private, the only way to set it is through the editor in edit mode. If you make these fields public, one could set them through code as well. Some fields that might change at runtime (e.g. lsv, lt, etc) that might be more desirable

Comment (victorhurdugaci) [reply UnitySDK/Assets/PlayerWon-SDK/Classes/ClientDetails.cs:65]: ps: hello :)

Comment (lauri-h) [reply UnitySDK/Assets/PlayerWon-SDK/Classes/ClientDetails.cs:65]: Intruder alert!

**Pull request merged**